src/components/bottom-sheet/bottom-sheet.tsx

```diff
@@ -99,11 +99,12 @@ export class BottomSheetComponent extends React.Component<
                             activeOpacity={1}
                         />
                         <DashboardMenuBottomSheet
+                            selectedAccount={this.props.selectedAccount}
```

Comment:
there is a selector getSelectedAccount - it returns the current account that you are on.
No need for external props but be careful cause selectedAccount will be undefined when the app is first installed

Reply:
I thought we try to not use redux in components. that's why I passed it as a prop